### ISSUE TITLE
Duplicate message creates unwanted info message while joining a match for watching

### ISSUE THREAD

**Issue by LevelX2** [magefree/mage#3207]
The message that is send if a user creates duplicate messages is also (unwanted) shown during the join to a match for watching.

Expected:
Such a message is not shown in this case.

![message](https://cloud.githubusercontent.com/assets/2066790/25303155/55d4262a-274d-11e7-974b-e53440b644fa.jpg)

**Comment by LevelX2** [comment]
Because the sound is also initiated from the chat message I guess the following problem that some sounds do no longer work is also caused somehow from chant changes.

09:39: Predator: I hate that the "player has joined the table" sound is missing 
09:41: Rybsson: people are now taking forever to start the game/tournament ;)
09:41: Rybsson: if they're alt-tabbed

**Comment by spjspj** [comment]
@LevelX2 - I realised I wasn't sure how to duplicate this scenario.  
>> Local Server, 3 dev clients connected to it:
I created a game, and joined with two of the clients.  I then started watching with the third one (more than once) but it didn't give me the 'You message appears to be identical to your last message'.

![image](https://cloud.githubusercontent.com/assets/16917728/25364993/70fb6392-29a9-11e7-9a15-4997a79c53d3.png)
